annotations/split_manifest_grants.py

```diff
@@ -0,0 +1,63 @@
+"""Split Manifests CSV
+This script will split a manifest csv by grant number and output
+results into individual CSVs.
+author: verena.chung
+author: brynn.zalmanek
+"""
+
+import os
+import argparse
+import pandas as pd
+
+
+def get_args():
+    """Set up command-line interface and get arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("manifest",
+                        type=str,
+                        help="path of manifest to be split")
+    parser.add_argument("manifest_type",
+                        type=str,
+                        choices=["publication", "dataset", "tool", "project"],
+                        help="type of manifest to split, e.g. publicaiton")
+    parser.add_argument("folder",
+                        type=str,
+                        help="folder path to save split manifests in")
+    return parser.parse_args()
+
+
+def get_df(manifest_csv):
+    """Convert manifest to data frame."""
+    df = pd.read_csv(manifest_csv)
+    return (df)
+
+
+def split_manifest(df, type, directory):
+    """Split manifest into multiple manifests by grant number"""
+    colname = f"{type.capitalize()} Grant Number"
+
+    df[colname] = df[colname].str.split(', ')
```

Comment:
Inconsistent use of `"` and `'` - try to stick with one. (not that I'm great at always sticking to this convention either 😅 )

I usually use `"` for all things strings and `'` for things like indexes, keys, etc.  Here's a [good example](https://stackoverflow.com/a/56190).

Reply:
This is good to know! I think I fixed all of them.